mds/access_control/conf_check.py

```diff
@@ -0,0 +1,15 @@
+from mds.server.settings import AUTH_MEANS
+
+
+def conf_check_middleware(get_response):
+
+    if not AUTH_MEANS:
+        raise Exception(
+            "JWT authentication configuration is incomplete: "
+            + "neither secret nor public key found"
+        )
+
+    def middleware(request):
+        return get_response(request)
+
+    return middleware
```

Comment:
The middleware will check conf on every request, that seems a little bit overkill

Reply:
I agree, I will just move the check when receiving a request (not optimal...)